src/sentry/utils/eap.py

```diff
@@ -0,0 +1 @@
+EAP_ITEMS_INSERT_ENDPOINT = "/tests/entities/eap_items/insert_bytes"
```

Comment:
This utils file will be further populated by https://github.com/getsentry/sentry/pull/103182